share/nisar/defaults/gcov.yaml

```
@@ -134,6 +134,11 @@ runconfig:
             # the corresponding metadata field will be set to "(NOT SPECIFIED)".
             product_doi:
 
+            # Data access for the static layers product associated with the
+            # output product (URL or DOI). If not provided, the corresponding
+            # metadata field will be set to "(NOT SPECIFIED)".
```

Let's complete this comment by making the remaining nominal behavior explicit:

```suggestion
            # metadata field will be set to "(NOT SPECIFIED)".
            # If this string contains the substring "{granule_id}", that substring
            # will be replaced with the granule ID.
```

---

Thank you, @nemo794 , these comments were updated in the commit [aa218dc](https://github.com/isce-framework/isce3/pull/100/commits/aa218dc90fa9f58f8feaa58cd757b3f18593841d).